File: src/python/maps/map12.py
def to11(engineRef):
    yield from engineRef.mapSwitchTask('map11.ika-map', (23 * 16, 17 * 16))

# Unused?
def heal(engineRef):
    engineRef.player.stats.hp = 999
    engineRef.player.stats.mp = 999
    if False:
        yield None

File: src/winter/maps/map12.ts
import { MapScript } from "./mapscript.js"
import { PyEngine } from "./../winter.js"

export default new MapScript(autoexec, { to11 })

export function autoexec(_engineRef: PyEngine) {}

export function *to11(engineRef: PyEngine) {
    yield* engineRef.mapSwitchTask('map11.ika-map', [23 * 16, 17 * 16])
}
